app/src/main/java/com/alphawallet/app/ui/widget/holder/WalletSummaryHolder.java

```diff
@@ -45,35 +53,62 @@ public class WalletSummaryHolder extends BinderViewHolder<Wallet> implements Vie
     private final ImageView defaultWalletIndicator;
     private final ImageView manageWalletBtn;
     private final UserAvatar walletIcon;
-    private final LinearLayout walletClickLayout;
+    private final RelativeLayout arrow_right;
     private final TextView walletBalanceText;
     private final TextView walletNameText;
     private final TextView walletAddressSeparator;
     private final TextView walletAddressText;
     private final TextView wallet24hChange;
+    private final RecyclerView recyclerView;
     private final Realm realm;
     private RealmResults<RealmWalletData> realmUpdate;
-
     private final WalletClickCallback clickCallback;
     private Wallet wallet = null;
+    protected final MutableLiveData<ErrorEnvelope> error = new MutableLiveData<>();
+    protected Disposable disposable;
+    private final Context context;
 
-    public WalletSummaryHolder(int resId, ViewGroup parent, WalletClickCallback callback, Realm realm)
+    public WalletSummaryHolder(int resId, ViewGroup parent, WalletClickCallback callback, Realm realm, Context context)
     {
         super(resId, parent);
+
         defaultWalletIndicator = findViewById(R.id.image_default_indicator);
         manageWalletBtn = findViewById(R.id.manage_wallet_btn);
         walletIcon = findViewById(R.id.wallet_icon);
         walletBalanceText = findViewById(R.id.wallet_balance);
         walletNameText = findViewById(R.id.wallet_name);
         walletAddressSeparator = findViewById(R.id.wallet_address_separator);
         walletAddressText = findViewById(R.id.wallet_address);
-        walletClickLayout = findViewById(R.id.wallet_click_layer);
+        arrow_right = findViewById(R.id.container);
         wallet24hChange = findViewById(R.id.wallet_24h_change);
+        recyclerView = findViewById(R.id.horizontal_list);
         clickCallback = callback;
+        this.context = context;
+
         manageWalletLayout = findViewById(R.id.layout_manage_wallet);
         this.realm = realm;
     }
 
```

Comment:
Is the onError still used? If not it can be removed, along with the supporting Mutable